src/lib/input/input-container.spec.ts

```diff
@@ -706,6 +709,116 @@ describe('MdInputContainer', function () {
       });
     }));
 
+    it('should display an error message when a custom error matcher returns true', async(() => {
+      fixture.destroy();
```

Comment:
can we just put this in a different describe block or something? its weird that we destroy the original fixture here